ice-phishing-detection-bot/src/agent.config.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+  getAPI: (chainId) => {
+    switch (chainId) {
+      case 1:
+        return {
+          APIKey: "API_KEY_HERE",
```

Comment:
i recommend to put API KEYS in a separate file and add to gitignore, so it doesnt get accidentally committed